src/components/MessageCard/components/MessageCard.Survey.css.js

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components'
+import { getColor } from '../../../styles/utilities/color'
+
+export const SurveyUI = styled('div')`
+  background: ${getColor('grey.200')};
+  border-radius: 5px;
+  margin: 0px -16px -15px -16px;
```

Comment:
The padding and negative margins are so that the div stretches to almost the edge of the container. There's a very light grey background on this div which can be really hard to see. I made it pink here so that it's easier to see why the spacing is needed:

<img width="337" alt="CleanShot 2022-02-01 at 10 39 09@2x" src="https://user-images.githubusercontent.com/44473/152030555-02d25637-09ca-4293-a552-7b3d45039b0a.png">

And this is without the margin and padding:

<img width="335" alt="CleanShot 2022-02-01 at 10 41 49@2x" src="https://user-images.githubusercontent.com/44473/152030659-7761dee4-8977-4b2c-adc6-55ce943c3bc9.png">

Reply:
Ah, my display is really bad at distinguishing different shades of grey and white :smile: Thanks for the explanation! (and I of course have not noticed there's a `background` property...)